app/models/service_ansible_playbook.rb

```diff
@@ -47,6 +47,15 @@ def postprocess(action)
 
   private
 
+  def manageiq_extra_vars
+    {
+      'miq_url'       => MiqRegion.my_region.remote_ws_url,
+      'miq_token'     => Api::UserTokenService.new.generate_token(evm_owner.userid, 'api'),
+      'miq_service'   => href_slug,
+      'miq_requestor' => evm_owner.href_slug
```

Comment:
Since these are already under the 'manageiq' key I would say we can drop the `miq_` prefix.

Suggested key names:
```
api_url
api_token
service
user
```
Open to other suggestions.